cmd/vela-worker/server.go

```diff
@@ -37,23 +38,25 @@ func (w *Worker) server() error {
 	// https://pkg.go.dev/github.com/sirupsen/logrus?tab=doc#Tracef
 	logrus.Tracef("serving traffic on %s", w.Config.API.Port)
 
-	// if provided, start serving traffic with TLS on the provided worker port
-	//
-	// https://pkg.go.dev/github.com/gin-gonic/gin?tab=doc#Engine.RunTLS
-	if len(w.Config.Certificate.Cert) > 0 {
-		// validate cert files exists at the provided paths
-		_, err := os.Stat(w.Config.Certificate.Cert)
-		if err != nil {
-			logrus.Fatalf("Expecting certificate file at %s, got %v", w.Config.Certificate.Cert, err)
-		}
-		_, err = os.Stat(w.Config.Certificate.Key)
-		if err != nil {
-			logrus.Fatalf("Expecting certificate key at %s, got %v", w.Config.Certificate.Key, err)
+	// if running with HTTPS, check certs are provided and run with TLS.
+	if strings.EqualFold(w.Config.API.Protocol, "https") {
+		if len(w.Config.Certificate.Cert) < 0 && len(w.Config.Certificate.Key) < 0 {
```

Comment:
I think you have your `<` backwards here? Looks like it'll always equate to `if false && false` unless I'm reading it wrong :)

Reply:
Sure do. Forgot to change that back after testing something. Good catch!